client/src/main/scala/com/dwolla/cloudflare/domain/model/logpush/LogpushJob.scala

```diff
@@ -0,0 +1,20 @@
+package com.dwolla.cloudflare.domain.model.logpush
+
+case class LogpushJob(
+  id: Int,
+  enabled: Boolean,
+  name: Option[String],
+  logpullOptions: Option[String],
+  destinationConf: String,
```

Comment:
My concern with a codec is maintainability. Cloudflare supports S3 and GCP now, but could expand this or add new parameters to either. When this happens, I wouldn't want to wait for updates and pull in a new version in order to utilize it.

Reply:
One way to address this would be to include a passthru codec for the `LogpushDestination` tagged string in addition to codecs for more specific config classes. That would allow callers to use the typed config if they wanted, but leave the raw format available if the typed config didn't support what they wanted yet.

With the tagged type in place now, I think we could probably add this later on without a source breaking change using a typeclass, e.g.:

```scala
case class LogpushJob[T : LogpushDestinationCodec](…
  destinationConf: T,
  …
)
```

and as long as there was a `LogpushDestinationCodec[LogpushDestination]` instance in implicit scope, it should compile.

Reply:
If you don't object, I'd rather leave it as a string. Using an SDK shouldn't preclude you from reading API docs and understanding how it works. Otherwise, where do you draw the line? Do we create a codec for `logpull_options` and keep it up-to-date too?

Reply:
I would probably do that, but I understand not wanting to do so now. Using the tagged string type is a compromise that allows us to add it later if we want to without breaking anyone implementing with the raw string.